crates/bevy_ecs/src/storage/blob_vec.rs

```diff
@@ -89,12 +89,22 @@ impl BlobVec {
     /// `index` must be in bounds
     /// Allows aliased mutable access to `index`'s data. Caller must ensure this does not happen
     #[inline]
-    pub unsafe fn set_unchecked(&self, index: usize, value: *mut u8) {
+    pub unsafe fn initialize_unchecked(&mut self, index: usize, value: *mut u8) {
         debug_assert!(index < self.len());
         let ptr = self.get_unchecked(index);
         std::ptr::copy_nonoverlapping(value, ptr, self.item_layout.size());
     }
 
+    /// # Safety
+    /// index must be in-bounds
+    ///
```

Comment:
```suggestion
    ///
    /// index must be in-bounds
```

Reply:
> Allows aliased mutable access to `index`'s data

I'm not sure, does it? I would even say that `initialize_unchecked` doesn't allow that as well. Both take `&mut self` and don't return anything. 

Any possible problem with aliasing here can only happen due to other unsafe means, like calling `get_unchecked`, dereferencing the returned pointer and holding onto it.

Reply:
> I would even say that initialize_unchecked doesn't allow that as well. Both take &mut self and don't return anything.

I didn't notice that change. Please update the comment for `initialize_unchecked`.